eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -100,6 +103,8 @@ steps:
       -ClientSecret '$(opensource-aad-secret)' 
   displayName: Apply Documentation Updates
 
+- ${{ parameters.AdditionalSteps }}
```

Comment:
Is this valid pipelines yaml when the object is an empty array?

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1385742&view=results
This is what it looks like for empty list: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1385742&view=results